HEN_HOUSE/user_codes/egs_chamber/egs_chamber.cpp

```diff
@@ -108,22 +108,38 @@ extern __extc__ void F77_OBJ(photo,PHOTO)();
                 the_stack->latch[np1] = the_stack->latch[np]; \
                 the_stack->ir[np1] = the_stack->ir[np]; \
                 the_stack->dnear[np1] = the_stack->dnear[np]; \
+                the_extra_stack->nbr_splitting[np1] = the_extra_stack->nbr_splitting[np]; \
+
+// Extra stack to hold variance reduction data per particle
+struct EGS_ExtraStack {
+    int nbr_splitting[MXSTACK];
+    int nbr_splittingI;
```

Comment:
The `nbr_splittingI` is not used; leave it there nevertheless?

Reply:
It is used in `TRANSFERPROPERTIESTO#FROM#` on the mortran side, but indeed I tried removing it and it seems to both compile and run. I just copied what was done in cavity, but I'm pretty sure it can be removed.